### DISCUSSION TITLE
How to just show conflicts only?

### DISCUSSION THREAD

**Discussion by silverwolf-x** [tox-dev/pipdeptree#401] [Q&A] [answered]
How to just show conflicts only instead of both conflicts and all dependences trees? I just care about what the conflicts are, and the output is too long to see conflicts at once.

**Comment by kemzeb** [comment, marked as answer]
We don't expose an option to hide the dependency trees, but we do **write warning messages to stderr.** So, assuming you are using bash, you can do the following to avoid writing the dependency trees to stdout:
```bash
 pipdeptree > /dev/null
```

**Comment by monperrus** [comment]
related questions, it that possible to compute the conflicts from an external requirements.txt file, without installing it?

    $ pipdevtree -r requirements.txt

**Comment by kemzeb** [comment]
Currently this is not possible. The way it works is that we scan your environment for package metadata, and then snatch the installed version metadata to check if it does not match the [version specifier](https://packaging.python.org/en/latest/specifications/version-specifiers/#id5) associated to the requirements that share the same package name as the installed one.

**Comment by monperrus** [comment]
thanks for the answer @kemzeb. do you know a tool which would do so?
